packages/next/lib/eslint/writeDefaultConfig.ts

```diff
@@ -51,10 +52,10 @@ export async function writeDefaultConfig(
     )
   } else if (!exists) {
     await fs.writeFile(
-      '.eslintrc.json',
+      path.join(baseDir, '.eslintrc.json'),
       CommentJson.stringify(selectedConfig, null, 2) + os.EOL
     )
-
+    console.log(path.join(baseDir, '.eslintrc.json'))
```

Comment:
Did you intend to keep this console log here?

Reply:
Oops was only in there for testing/debugging. Removed 😅